packages/optivum-scrapper/src/table.ts

```diff
@@ -30,45 +30,34 @@ export class Table {
         niedziela: 7,
     };
 
-    public getFullName(): string | undefined {
-        return /Plan lekcji (?:oddziału|nauczyciela|sali) - (.+)/.exec(this.document.title)?.[1];
-    }
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    public getFullName = (): string => this.document.querySelector('span.tytulnapis')!.textContent!;
```

Comment:
Nie masz podstaw, żeby założyć że `span.tytulnapis` zawsze istnieje, a `!` nie daje żadnych runtime checków. Możesz zrobić:
```suggestion
    public getFullName = (): string => this.document.querySelector('span.tytulnapis')?.textContent ?? '';  
```